### PR TITLE
Added the BDD scenario with locators for Homepage and ProductDisplayPage class

### PR DESCRIPTION
**Pull request by ashivananjappa-GS**

### CODE REVIEW

File: src/main/java/session15locators/tasks/akshatha/ProductDisplayPage.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+  private static final String XXS_SIZE_BUTTON = "pdp-size-xxs-select";
+  private static final String XS_SIZE_BUTTON = "pdp-size-xs-select";
+  private static final String S_SIZE_BUTTON = "pdp-size-s-select";
+  private static final String L_SIZE_BUTTON = "pdp-size-l-select";
+  private static final String XL_SIZE_BUTTON = "pdp-size-xl-select";
+  private static final String XXL_SIZE_BUTTON = "pdp-size-xxl-select";
+
+  private static final String ADD_TO_BAG_BUTTON = "pdp-addToBag-submit";
+  private static final String ADD_TO_WISHLIST_ICON = "pdp-addToWishlist-ad";
```
Comment (dwilliams-gs) [inline src/main/java/session15locators/tasks/akshatha/ProductDisplayPage.java:13]: Small typo `add`

File: src/main/java/session14bddgherkin/tasks/akshatha/Wishlist.feature [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Feature: Wishlist Functionality
+
+  Scenario: Adding an item to the wishlist
+    Given User is logged in
+    And User is on the PDP screen
+    When User adds an item to the wishlist
+    Then Item is added to their Wishlist
+    And Confirmation message is displayed
```
Comment (dwilliams-gs) [inline src/main/java/session14bddgherkin/tasks/akshatha/Wishlist.feature:8]: If interested take a look at `@Wishlist-002` to see our current BDD for this test.

**Review by dwilliams-gs — APPROVED** [review]
LGTM 👍  Small type on locator

File: src/main/java/session14bddgherkin/tasks/akshatha/Wishlist.feature (hunk 2 above)
Comment (ashivananjappa-GS) [reply src/main/java/session14bddgherkin/tasks/akshatha/Wishlist.feature]: Had a look and updated the test slightly Thanks

File: src/main/java/session15locators/tasks/akshatha/ProductDisplayPage.java (hunk 1 above)
Comment (ashivananjappa-GS) [reply src/main/java/session15locators/tasks/akshatha/ProductDisplayPage.java]: Resolved

**Review by dwilliams-gs — APPROVED** [review]
LGTM 👍

**Pull request merged**